karrot/subscriptions/receivers.py

```diff
@@ -642,3 +642,19 @@ def feedback_saved(sender, instance, created, **kwargs):
 
     for subscription in ChannelSubscription.objects.recent().filter(user=user):
         send_in_channel(subscription.reply_channel, topic='status', payload=payload)
+
+
+@receiver(post_save, sender=ActivityParticipant)
+def activity_participant_saved(sender, instance, **kwargs):
```

Comment:
It would seem worth the small abstraction to me to implement a function: `send_feedback_possible_count(user)` to save the code duplication.

That can be called from the existing handlers `feedback_saved` and `send_activity_participant_updates` (which also handles the case of leaving the activity).

Reply:
tl;dr: nothing to see here! just Nick thinking too much :thinking: 

I was thinking that this still doesn't handle the case of leaving an activity and updating the count... but you can never leave an activity that it's possible to give feedback to (leaving is only before the activity, and feedback is only after the activity), so I think it's fine. There could be some nuance for the _during_ the activity time, we have some issues with that already, but yeah, don't need to think about them here.